docusaurus/docs/nested-plugins.mdx

```diff
@@ -3,7 +3,11 @@ id: nested-plugins
 title: Nested Plugins
 ---
 
-Grafana app plugins can nest frontend datasource and panel plugins making it easy to ship a complete user experience. To take advantage of this feature first scaffold an app plugin using `@grafana/create-plugin`:
+Grafana app plugins can nest frontend data sources together with panel plugins so that you can provide a complete user experience.
```

Comment:
`can nest frontend data sources` also backend ones. The original text is incorrect here. Can you remove the frontend part from this sentence please? `can nest data sources`

Reply:
removed